lib/fastlane/actions/ipa.rb

```diff
@@ -74,10 +74,23 @@ def self.run(params)
         ENV[SharedValues::DSYM_OUTPUT_PATH.to_s] = absolute_dsym_path
       end
 
-      def self.params_to_build_args(params)
-        # Remove nil value params unless :clean or :archive
-        params = params.delete_if { |k, v| (k != :clean && k != :archive) && v.nil? }
-
+      def self.params_to_build_args(config)
+        params = {}
+        params[:workspace] = config[:workspace]
```

Comment:
If you pull from `fastlane_core`, you can now replace all that with
```ruby
params = config.values
```

:rocket: